research/query_service/ir/runtime/src/graph/mod.rs

```diff
@@ -74,7 +75,7 @@ pub struct QueryParams {
     pub columns: Option<Vec<NameOrId>>,
     pub partitions: Option<Vec<u64>>,
     pub filter: Option<Arc<PEvaluator>>,
-    pub extra_params: Option<HashMap<String, String>>,
+    pub extra_params: Option<HashMap<String, Object>>,
```

Comment:
This does not need to change now, if you donot store PK here?

Reply:
I have reverted this.